source/index.html.md

```diff
@@ -160,6 +160,21 @@ Exceeding the limit will yield an empty response with status code `429`.
 **Note: We continue to count any API calls made while the limit is exceeded.**
 
 
+# Scoping
+
+Scopes are defined at an OAuth App level. The below table describes the scopes and the corresponding GraphQL queries/mutation that they allow.
+
+Scope | Queries | Mutations
+----- | ------- | ---------
+`user::flow:read` | `flow`, `flows`, `flows.steps`, `findAddableFlowsForPerson` |
```

Comment:
Maybe a description would be useful? I know Slack has a page dedicated to what the scopes mean. 

![image](https://user-images.githubusercontent.com/8136030/56595703-f8d08300-65a3-11e9-8c45-5e90243617ec.png)

Also, maybe _some_ background on the general structure of a scope string (what each part means, for example) might be useful to integrators.